### DISCUSSION TITLE
i'm having a problem with opengl

### DISCUSSION THREAD

**Discussion by wakkusu16** [raysan5/raylib#2313] [Q&A] [answered]
Yesterday it was working perfectly, but today when I went to open my exe file, this error popped up:
`WARNING: GLFW: Error: 65542 Description: WGL: The driver does not appear to support OpenGL raylib`
can someone help me?

**Comment by jamesl-github** [comment, marked as answer]
update drivers ?

https://www.minecraftstation.com/how-to-fix-glfw-error-65542-in-minecraft/

run on dedicated gpu and not integrated if possible ?

**Reply by wakkusu16** [reply]
Thanks for the help but, it didnt worked, so i just reseted my entire computer and it works now
